apps/minaintyg/src/schema/certificate.schema.ts

```diff
@@ -12,9 +12,8 @@ export const certificateSummarySchema = z.object({ label: z.string(), value: z.s
 
 export const certificateTypeSchema = z.object({
   id: z.string(),
-  name: z.string(),
```

Comment:
Since we use the same metadata for both list certificate and read certificate in frontend we need to keep name. It will be used to display the name of the certificate in the list certificate view.

An alternative solution could be that we separate the two different metadata.

Reply:
I moved the name from metadata to type for certificate response using `transformResponse` so that it follows the same schema.